### PR TITLE
[Data-Plane Mgr] Fix DPM NPE and gRPC issues.

### PR DESCRIPTION
**Pull request by cj-chung**

### CODE REVIEW

File: services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public InternalDPMResultList createNetworkConfiguration(@RequestBody NetworkConf
         return dpmService.createNetworkConfiguration(networkConfiguration);
     }
 
-    @PostMapping({"/network-configuration", "v4/network-configuration"})
+    @PutMapping({"/network-configuration", "v4/network-configuration"})
```
Comment (xieus) [inline services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java:48]: Do we revert the change in https://github.com/futurewei-cloud/alcor/commit/971f0b602c5dbe645a28ca5f8659ffc7d7b24e7a?

Comment (cj-chung) [reply services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java:48]: No, I think I reuse my branch for this PR.

**Review by xieus — APPROVED** [review]
LGTM.
